tests/bluetooth/tester/src/mesh.c

```diff
@@ -2045,6 +2051,214 @@ static void config_lpn_timeout_get(uint8_t *data, uint16_t len)
 		   err ? BTP_STATUS_FAILED : BTP_STATUS_SUCCESS);
 }
 
+static void health_fault_get(uint8_t *data, uint16_t len)
+{
+	struct mesh_health_fault_get_cmd *cmd = (void *)data;
+	uint8_t test_id;
+	uint8_t faults;
+	size_t fault_count;
+	int err;
+
+	LOG_DBG("");
+
+	err = bt_mesh_health_fault_get(cmd->address, cmd->app_idx, cmd->cid,
+				       &test_id, &faults, &fault_count);
+
+	if (err) {
+		LOG_ERR("err %d", err);
+		goto fail;
+	}
+
+fail:
+	tester_rsp(BTP_SERVICE_ID_MESH, MESH_HEALTH_FAULT_GET, CONTROLLER_INDEX,
+		   err ? BTP_STATUS_FAILED : BTP_STATUS_SUCCESS);
+}
+
+static void health_fault_clear(uint8_t *data, uint16_t len)
+{
+	struct mesh_health_fault_clear_cmd *cmd = (void *)data;
+	uint8_t test_id;
+	uint8_t faults;
+	size_t fault_count;
```

Comment:
`faults` should be an array and `fault_count` should contain the number of faults that the application can handle (so the size of the array).